### PR TITLE
feature/#48-add-check-view-page

### PR DESCRIPTION
**Pull request by AlejandroSalcido**
I created the basic frontend of the Check page, please review it for future enhancements

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
Please make the desktop view too as part of this ticket

image

I only notice mobile view

**Comment by Israel-Laguan** [comment]
Please wait until input and form is done @AlejandroSalcido

**Comment by AlejandroSalcido** [comment]
> Please wait until input and form is done @AlejandroSalcido

Currently im starting to work on those ones, i would add them later as components when both are finished

**Comment by ginnaaceror** [comment]
@AlejandroSalcido I can see the responsive its ok, It would be good to modularize the part of styles components.

File: src/pages/Check.jsx [hunk 1]
```diff
@@ -1,17 +1,148 @@
...
+      <BackButton src={arrow} alt="Arrow left" />
+      <Heading>Hola!</Heading>
+      <Text>Ingresa tu numero de boleta</Text>
+      <TicketImage src={ticket} alt="Ticket" />
+    </Header>
+    {/* <div className="pop-up">
+        <img src={warning} alt="Warning" />
+      </div> */}
+    <MainSection>
+      <MainText>CONFIRMAR Nº DE BOLETA O PEDIDO</MainText>
```
Comment (ginnaaceror) [inline src/pages/Check.jsx]: I would use "text-transform: uppercase" property in the css and the html text in lowercase.

**Review by ginnaaceror — COMMENTED** [review]
I like the tags names. I can understand the reference you make :)

**Pull request merged**